bridge/client/app/_views/ktb-sequence-view/ktb-sequence-view.component.html

```diff
@@ -97,15 +105,8 @@
           <p class="m-0 mb-1">
```

Comment:
Please remove this part of the code - so the whole `<p>` element with children.

Reply:
Removed here: 59d62607210a04df5f9857cdb008a8e84422ecbe. Thanks!